pkg/deploy/registry/deploylog/rest.go

```diff
@@ -25,8 +26,12 @@ import (
 	deployutil "github.com/openshift/origin/pkg/deploy/util"
 )
 
-// defaultTimeout is the default time to wait for the logs of a deployment
-const defaultTimeout time.Duration = 20 * time.Second
+const (
+	// defaultTimeout is the default time to wait for the logs of a deployment.
+	defaultTimeout time.Duration = 20 * time.Second
```

Comment:
Hrm.  Maybe.  The client will actually retry ServerTimeout in most cases.

On Jun 29, 2016, at 3:33 PM, Michail Kargakis <notifications@github.com>
wrote:

In pkg/deploy/registry/deploylog/rest.go
<https://github.com/openshift/origin/pull/9482#discussion_r69040670>:

> @@ -25,8 +26,12 @@ import (
>  	deployutil "github.com/openshift/origin/pkg/deploy/util"
>  )
>
> -// defaultTimeout is the default time to wait for the logs of a deployment
> -const defaultTimeout time.Duration = 20 * time.Second
> +const (
> +	// defaultTimeout is the default time to wait for the logs of a deployment.
> +	defaultTimeout time.Duration = 20 * time.Second

what error code we return when this timeout is reached?

Depends on where do we reach it. If we timeout while waiting for the
replication controller to be created, we return a notFound error because we
were polling on that error. If we reach this while waiting for the
replication controller to become running then we return a timeout error.

can client retry getting the logs based on the return code?

It's possible. Do we want to do something for oc?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/9482/files/53571d0332a00a9aa0df0515d34a867b65273f42#r69040670>,
or mute the thread
<https://github.com/notifications/unsubscribe/ABG_p2fpUwlnuRCLLuZW7te7tmbk7kN-ks5qQvKlgaJpZM4I7sgr>
.